### PR TITLE
2 exercises added

### PR DESCRIPTION
**Pull request by smddzcy**
TODO: Add an **algorithm exercise** and an **old code to functional code conversion** exercise. (maybe more?)

### CODE REVIEW

**Comment by omurturan** [comment]
@smddzcy are you done with this pull request?

**Comment by smddzcy** [comment]
@omurturan i'll add more exercises but yes, you can merge it.

File: FE101-Exercises/02-Top-Frequent-Elements/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+```javascript
+var res = [1, 1, 1, 2, 2, 3].topFrequent(2);
+console.log(res); // [1, 2]
+```
+
+***
+
+**Note:**
+
+* Your code's time complexity *must not* exceed `O(n log n)`.
```
Comment (mikesoylu) [inline FE101-Exercises/02-Top-Frequent-Elements/README.md:29]: @smddzcy will this be too advanced? maybe we can talk about big o notation in one of the intro courses

File: FE101-Exercises/03-Closures/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Specs
+
+Consider the following code:
+
+```javascript
+var nodes = document.getElementsByTagName('button');
+for (var i = 0; i < nodes.length; i++) {
+   nodes[i].addEventListener('click', () => {
```
Comment (mikesoylu) [inline FE101-Exercises/03-Closures/README.md]: indentation is 3 spaces

**Comment by mikesoylu** [comment]
@smddzcy really good thx 👍👍

File: FE101-Exercises/03-Closures/README.md (hunk 2 above)
Comment (smddzcy) [inline FE101-Exercises/03-Closures/README.md]: fixed, thanks for noticing 😄

**Pull request merged**